Applying split function on a NON string variable

Guys, I've gotten this error: ```i.split is not a function``` while clicking on map created by [leaflet-directive](https://github.com/tombatossals/angular-leaflet-directive).

The line code can be found here: https://github.com/driftyco/ionic-platform-web-client/blob/master/dist/ionic.io.bundle.js#L2499
```javascript
        var className = element.className;
        if (className) {
          var classes = className.split(' ');
          for (var i = 0; i < classes.length; i++) {
            var c = classes[i];
            if (c) {
              selector += '.' + c;
            }
          }
        }
```

I logged the variables in order to know what kind of object split function was receiving. My log:
```
element: {"_leaflet_pos":{"x":-180,"y":-241},"_simulatedClick":true} 
classname: {}
type: object
```

The code check if `className` exists. As you can see, the `className` exists, however, it is an empty object. So, when trying to split, the error is thrown.

**My suggestion is to check if `className` is a string before trying to split it.**

p.s.: I think this is related to Ionic Analytics, because I got the log print on click, whatever is the place. When the click occurs on Ionic buttons for instance, I get the following log: 
```
element: {"$$hashKey":"object:43"} 
classname: "icon ion-ios-person-outline" 
type: string
```

--- a/src/analytics/serializers.js
+++ b/src/analytics/serializers.js
@@ -13,7 +13,7 @@ export class DOMSerializer {
 
       var className = element.className;
       if (className) {
-        var classes = className.split(' ');
+        var classes = ('' + className).split(' ');
         for (var i = 0; i < classes.length; i++) {
           var c = classes[i];
           if (c) {

